test-result-summary-client/src/Dashboard/Widgets/Graph/ODM.jsx

```diff
@@ -96,12 +95,21 @@ export default class ODM extends Component {
                 if(jdkDate) {
                     resultsByJDKBuild[t.buildName][jdkDate] = resultsByJDKBuild[jdkDate] || [];
                     resultsByJDKBuild[t.buildName][jdkDate].push( {
-                        globalThroughput: t.tests[0].testData.metrics[0].value[0],
+                        globalThroughput: math.round(aggregateInfo.metrics[0].value["mean"], 3),

```

Comment:
Same issue as above. It doesn't have to be index 0.

Reply:
Maybe we should just store rounded off values in the backend itself so that we don't need to round off every time in different tools. In the backend, we should only round off right at the end to reduce error margins in calculations.